File: app/src/main/java/com/synapse/social/studioasinc/widget/ZoomImageViewLib/ZoomInImageView.java
package com.synapse.social.studioasinc.widget.ZoomImageViewLib.gestures;

import android.content.Context;
import android.util.AttributeSet;
import android.widget.ImageView;

public class ZoomInImageView extends ImageView {
    ZoomInImageViewAttacher attacher;

    public ZoomInImageView(Context context) {
        this(context, (AttributeSet) null);
    }

    public ZoomInImageView(Context context, AttributeSet attrs) {
        this(context, attrs, 0);
    }

    public ZoomInImageView(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
        this.attacher = new ZoomInImageViewAttacher(this);
    }
}

File: .backup/additional_orphaned_20260101_212328/ZoomInImageView.kt
package com.synapse.social.studioasinc.widget.ZoomImageViewLib

import android.content.Context
import android.util.AttributeSet
import androidx.appcompat.widget.AppCompatImageView

class ZoomInImageView @JvmOverloads constructor(
    context: Context,
    attrs: AttributeSet? = null,
    defStyleAttr: Int = 0
) : AppCompatImageView(context, attrs, defStyleAttr) {

    private val attacher: ZoomInImageViewAttacher = ZoomInImageViewAttacher(this)

}
